specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -300,6 +300,88 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{resourceName}/listresourcepreparationerrors": {
+      "post": {
```

Comment:
What's the reason this apparent read operation is a POST method? Are the response data privileged? If not, it would be preferred to model as a GET to a proxy resource (e.g. `/workspaces/resourcePreparationErrors`)

Reply:
The resourcePreparationErrors doesn't support Create/Update/Delete from user, it's just telling the customer what the errors are for the resource preparation. That case, should I still model that as a GET to a proxy resource?

Reply:
changed from post to get